src/modules/loadCharacters.js

```diff
@@ -32,11 +32,81 @@ export async function loadCharacters() {
           </div>
         </div>
         <div class="reactions">
-          <span class="comment">Comments</span>
+          <span class="comment" data-id="${character.id}" data-bs-toggle="modal" data-bs-target="#modal-container">Comments</span>
           <span class="reservation">Reservations</span>
         </div>
       `;
 
     charactersContainer.appendChild(characterElement);
   });
+
+  return characters; // Return the characters array
+}
+
+// Function to fetch additional character details
+export async function fetchCharacterDetails(characterId) {
+  try {
+    const response = await fetch(`https://rickandmortyapi.com/api/character/${characterId}`);
+    const data = await response.json();
+    return {
+      image: data.image,
+      name: data.name,
+      status: data.status,
+      species: data.species,
```

Comment:
This module has been repeated in loadCharacters.js. I will recommend you delete one of the modules and maintain one since they seem to do the same thing